### PR TITLE
CR-11574

### PR DESCRIPTION
**Pull request by elad-codefresh**

### CODE REVIEW

File: cmd/commands/git-source.go [hunk 1]
```diff
@@ -524,6 +524,10 @@ func RunGitSourceList(ctx context.Context, runtimeName string, includeInternal b
 			continue
 		}
 
+		if gs.Self == nil {
```
Comment (roi-codefresh) [inline cmd/commands/git-source.go:527]: This means you will not show it at all, I wonder if this is what we want. How does UI deal with this state?
We need to have the same behavior

Comment (elad-codefresh) [reply cmd/commands/git-source.go:527]: currently, in this situation (no self) the ui currently won't show it as well (with the recently-added descriptive error )

Comment (roi-codefresh) [reply cmd/commands/git-source.go:527]: Oh, but it does show an error, maybe we should also print some error

**Review by roi-codefresh — APPROVED** [review]

File: cmd/commands/git-source.go [hunk 2]
```diff
@@ -524,6 +524,12 @@ func RunGitSourceList(ctx context.Context, runtimeName string, includeInternal b
 			continue
 		}
 
+		if strings.Contains(name, "default") {
```
Comment (roi-codefresh) [inline cmd/commands/git-source.go]: This is a bug

**Review by roi-codefresh — CHANGES_REQUESTED** [review]

**Review by roi-codefresh — APPROVED** [review]

**Pull request merged**